<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDomainPF</class>
 <widget class="QMainWindow" name="frmDomainPF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>物理场</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxDomainPF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>求解域设置</string>
    </property>
    <widget class="QCheckBox" name="chkModel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>模型域</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkAir">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>空气域</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkPML">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>完美匹配层</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
